test/perf/benchmarks/assets/compiler.test.ps1

```diff
@@ -1203,7 +1203,7 @@ function Start-PSBootstrap {
                 }
             } elseif ($environment.IsLinux -and $environment.IsSUSEFamily) {
                 # Build tools
-                $Deps += "gcc", "cmake", "make"
+                $Deps += "gcc", "make"
```

Comment:
same

Reply:
ditto. changes to this file can be reverted.